CONTRIBUTING.md

```diff
@@ -166,7 +166,7 @@ The core Docusaurus team will be monitoring for pull requests. Do help us by kee
 
 ## Style Guide
 
-[Prettier](https://prettier.io) will catch most styling issues that may exist in your code. You can check the status of your code styling by simply running `npm run prettier`.
```

Comment:
Why are these changes in this PR? After all, it in no way relate to the purpose of PR.

Reply:
I figured it was okay since I called it out in the PR description, and because it's changing `npm run prettier` to this `yarn prettier` (two words essentially). If you'd really prefer, I can open a separate PR. I was trying to be efficient.

Reply:
Please make changes to PR only those that are directly related to the topic of PR.

The title PR now says “check repoUrl before using on users.js”, which means that changes to this PR should only apply to this bug fix.

The exception here is files that have been modified via git hooks, which is not very good, but acceptable

Reply:
Got it! I will revert the unrelated changes and a separate PR.